ts/src/index.ts

```diff
@@ -35,9 +35,15 @@ y.command(
         return;
       }
       const absolutePath = path.resolve(argv.path);
+      const files = fs.readdirSync(absolutePath);
+      const models: string[] = [];
+      files.forEach((file) => {
+        if (file !== "index.ts" && file.endsWith(".ts"))
+          models.push(absolutePath + "/" + file);
+      });
       const sequelize = new Sequelize({
         dialect: argv.dialect,
-        models: [absolutePath + "/*.ts"],
+        models: models,
```

Comment:
```suggestion
        models: files.reduce((models, file) => {
                if (file !== "index.ts" && file.endsWith(".ts")) {
                        models.push(absolutePath + "/" + file)
                }
                return models;
        }, []),
```

More JS'ish but just a suggestion.